flixel/addons/editors/pex/FlxPexParser.hx

```diff
@@ -54,7 +55,7 @@ class FlxPexParser
 		var angle = minMax("angle", config);
 		
 		var startSize = minMax("startParticleSize", config);
-		var finishSize = minMax("finishParticleSize", "FinishParticleSizeVariance", config);
+		var finishSize = minMax("finishParticleSize", "finishParticleSizeVariance", config);
```

Comment:
When I export the default particle effect on http://onebyonedesign.com/flash/particleeditor/, it's also capitalized for me.